recaf-ui/src/main/java/me/coley/recaf/ui/context/MethodContextBuilder.java

```diff
@@ -77,6 +81,7 @@ public ContextMenu build() {
 			menu.getItems().add(refactor);
 			Menu view = menu("menu.view", Icons.EYE);
 			view.getItems().add(action("menu.view.methodcfg", Icons.CHILDREN, this::graph));
```

Comment:
Merge note to self: Can make this more specific/clear now that there are two _"graph"_ options